src/components/studyList/StudyList.js

```diff
@@ -256,28 +257,28 @@ class StudyList extends Component {
   render() {
     const tableMeta = {
       patientName: {
-        displayText: 'Patient Name',
+        displayText: this.props.t('PatientName'),
         sort: 0,
       },
       patientId: {
-        displayText: 'MRN',
+        displayText: this.props.t('MRN'),
         sort: 0,
       },
       accessionNumber: {
-        displayText: 'Accession #',
+        displayText: this.props.t('Accession'),
```

Comment:
Can we make the key for this AccessionNumber instead? That will make it more clear for translators what this is for.

Reply:
Done